parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/state/ShardManager.java

```diff
@@ -78,13 +78,13 @@ public class ShardManager<K, V> {
      */
     @Getter(AccessLevel.PACKAGE) // visible for testing
     private final Comparator<WorkContainer<?, ?>> retryQueueWorkContainerComparator = Comparator
-            .comparing((WorkContainer<?, ?> workContainer) -> workContainer.getRetryDueAt())
-            .thenComparing(workContainer -> {
+            //.comparing((WorkContainer<?, ?> workContainer) -> workContainer.getRetryDueAt())
```

Comment:
cannot comment that out - it is used for ordering retry queue when picking work.